openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/EncounterResource.java

```diff
@@ -145,17 +137,35 @@ public Object createInner(SimpleObject post, RequestContext context) throws Resp
         if (encounter == null) {
             throw new InvalidObjectDataException("No observations specified");
         }
-        return encounterToJson(encounter);
+        SimpleObject simpleObject = new SimpleObject();
+        populateJsonProperties(encounter, context, simpleObject, 0);
+        return simpleObject;
     }
 
     /**
-     * Converts an encounter to its JSON representation, filling in observation data.
-     * @param encounter an encounter instance
-     * @return a SimpleObject representing the encounter, ready to be serialized to JSON
+     * Retrieves the encounter with a given UUID. Currently not implemented.
      */
-    private SimpleObject encounterToJson(Encounter encounter) {
-        SimpleObject encounterJson = new SimpleObject();
+    @Override
+    protected Encounter retrieveImpl(String uuid, RequestContext context, long snapshotTime) {
+        // Not implemented
+        return null;
+    }
 
+    /**
+     * Populates observation and order data for the given encounter, including:
+     * <ul>
+     *   <li>"timestamp": the encounter datetime in RFC 3339 === ISO 8601 format
+     *   <li>"uuid": the encounter's UUID
+     *   <li>"observations": {@link SimpleObject} that maps concept UUIDs to values
+     *   <li>"order_uuids": unique identifiers of orders related to this encounter.
```

Comment:
I've amended the comment to state that this is orders executed.
Thanks for the information re incremental order sync. It sounds complex and is certainly beyond what I understand at the moment, so I'll address that in a future PR.